account_reversal/account_reversal.py

```diff
@@ -129,26 +121,16 @@ def create_reversals(self, cr, uid, ids, reversal_date,
 
         :return: Returns a list of ids of the created reversal moves
         """
-        if isinstance(ids, (int, long)):
-            ids = [ids]
-
-        reversed_move_ids = []
-        for src_move in self.browse(cr, uid, ids, context=context):
-            if src_move.reversal_id:
-                continue  # skip the reversal creation if already done
-
-            reversal_move_id = self._move_reversal(
-                cr, uid,
-                src_move,
-                reversal_date,
-                reversal_period_id=reversal_period_id,
-                reversal_journal_id=reversal_journal_id,
-                move_prefix=move_prefix,
-                move_line_prefix=move_line_prefix,
-                context=context
-            )
 
-            if reversal_move_id:
-                reversed_move_ids.append(reversal_move_id)
+        if self.reversal_id:
```

Comment:
Looks like you only process a single move here. Do you need to apply @api.one on the method?